### DISCUSSION TITLE
做空 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#20] [Announcements]
# 做空 - forecho's Blog

引言
今年美股可谓一波三折，从去年 11 月份开始一直到现在都是震荡下行的趋势。这波熊市谁也不知道什么时候到头。今天我们来聊聊做空的基本知识，以及做空的风险。
什么是做空？
如果我们看好一只股票，预期它会涨，那么你买入它，等它涨了你就卖出，这种情况我们叫做多，也就是 Long。
股票有涨也就有跌，与之相反的，你不看好一只股票，预期它会跌，那么你买出它，等它跌了你就买入，这种情况我们叫做空，也就是 Short。
做空你先卖股票，跟做多买股票一样，做空直接卖就行了。就拿最近半导体行业需求下滑来讲，你认为最近半导体股票还会继续下跌，那么你可以卖出 100 AMD，卖出的股票就相当于你向券商借了 100 股 AMD 股票，借的股票是需要支付一定的利息的（具体看个股详情），当股票下跌的时候，你买入 100 股 AMD，会自动还券商 100 股。

https://blog.forecho.com/short.html

**Comment by macwxx** [comment]
做空100跌到1跌幅99%
做多1涨到2涨幅100%
喜欢A还是喜欢V

**Reply by forecho** [reply]
这就是很多人不愿意或者不喜欢做空的原因，但是有时候真的就需要做空。

**Reply by macwxx** [reply]
作为现货的套保。
